helm_deploy/values-dev.yaml

```diff
@@ -26,7 +26,8 @@ generic-service:
     ASSIGN-DEFAULT-REGION-TO-USERS-WITH-UNKNOWN-REGION: true
     SENTRY_ENVIRONMENT: dev
     CACHES_STAFFMEMBERS_EXPIRY-SECONDS: 20
-    DOMAIN-EVENTS_EMIT-ENABLED: true
+    DOMAIN-EVENTS_CAS1_EMIT-ENABLED: true
```

Comment:
This [commit](https://github.com/ministryofjustice/hmpps-approved-premises-api/pull/1004/commits/bfc8a27ed28439a408ef4d5e7fd457bbac3246f5) where we move cas1 events into a namespace is probably one for you to be aware of @pezholio